src/executables/javasp.cpp

```diff
@@ -211,7 +211,7 @@ main (int argc, char *argv[])
 	    goto exit;
 	  }
 
-	status = javasp_start_server (jsp_info, db_name, pathname.c_str ());
+	status = javasp_start_server (jsp_info, db_name, pathname);
```

Comment:
what is different pathname.c_str() and pathname?

Reply:
c_str() returns char* of std::string. the type of an argument of javasp_start_server is std::string&.
it is just a slip that causes creating new std::string from char* by the constructor.

Reply:
please check the windows build fail

Reply:
I fixed it, thank you.